diff --git a/dist/index.mjs b/dist/index.mjs
index 92bb2af8e2daf5e197f5fa4484cf2c3f12ba629f..270d66a6dc2b8fd2d3a25d7522de531b58fed16d 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -468,10 +468,13 @@ async function registerTriggers(tx, tables, idColumnMap, indexerId) {
       );
       await tx.execute(
         sql.raw(`
+          DO $$ BEGIN
           CREATE CONSTRAINT TRIGGER ${getReorgTriggerName(table, indexerId)}
           AFTER INSERT OR UPDATE OR DELETE ON ${table}
           DEFERRABLE INITIALLY DEFERRED
           FOR EACH ROW EXECUTE FUNCTION ${SCHEMA_NAME}.reorg_checkpoint('${table}', '${tableIdColumn}', '${indexerId}');
+          EXCEPTION WHEN duplicate_object THEN NULL;
+          END $$;
         `)
       );
     }
