diff --git a/node_modules/@apibara/plugin-drizzle/dist/index.mjs b/node_modules/@apibara/plugin-drizzle/dist/index.mjs
index 92bb2af..270d66a 100644
--- a/node_modules/@apibara/plugin-drizzle/dist/index.mjs
+++ b/node_modules/@apibara/plugin-drizzle/dist/index.mjs
@@ -468,10 +468,13 @@ async function registerTriggers(tx, tables, idColumnMap, indexerId) {
       );
       await tx.execute(
         sql.raw(`
+          DO $$ BEGIN
           CREATE CONSTRAINT TRIGGER ${getReorgTriggerName(table, indexerId)}
           AFTER INSERT OR UPDATE OR DELETE ON ${table}
           DEFERRABLE INITIALLY DEFERRED
           FOR EACH ROW EXECUTE FUNCTION ${SCHEMA_NAME}.reorg_checkpoint('${table}', '${tableIdColumn}', '${indexerId}');
+          EXCEPTION WHEN duplicate_object THEN NULL;
+          END $$;
         `)
       );
     }
